### ISSUE TITLE
Character Relationships System?

### ISSUE THREAD

**Issue by omegaschtroumpf** [omegaschtroumpf/Happy-New-Year#2]
# Will it work?

Two-dimensional array created in the room.  A square, one row & column for each character in room, including player.  Each row is how a character feels about other characters.  row 0 (player character is moot).  row 1 (first enemy) has these columns: 0 - how char 1 feels about player; 1 - how char 1 feels about char 1 / unused; 2 - how char 1 feels about char 2

## What needs doing
* :white_check_mark: build array
* ~~write scripts to update array based on events~~
* :white_large_square: place calls to the array update scripts at the end of events
 * :white_check_mark: obj_character_collision with obj_sword
 * :white_large_square: etc. still to be done
* :white_large_square: some other task

**Comment by yarmulglat** [comment]
Yeah it looks like this could be viable. Where did you find the commands ("white_large_square, etc.") to draw the boxes and handle headers, etc.?

**Comment by omegaschtroumpf** [comment]
:information_source: When you write a comment or click to edit, there's a link "Markdown supported" to the right of the write and preview tabs.  At the bottom of the markdown page there's a link to the emoji cheat sheet.
